experimental/dds/tree2/src/test/simple-tree/object.spec.ts

```diff
@@ -291,10 +292,12 @@ testObjectLike(tcs);
 
 describe("Object-like", () => {
 	describe("setting an invalid field", () => {
-		it("throws TypeError in strict mode", () => {
-			const root = getOldRoot(
-				makeOldSchema((_) => _.object("no fields", {})),
-				{},
+		// TODO: Restore original behavior for bare '_.object()'?
+		// https://dev.azure.com/fluidframework/internal/_workitems/edit/6549
+		it.skip("throws TypeError in strict mode", () => {
+			const root = getRoot(
+				makeSchema((_) => _.object("no fields", {})),
+				() => ({}),
```

Comment:
'schema-class' allow the user to write fields that are not part of the schema.

These fields are not synchronized or persisted, which can be a feature or a bug depending on the circumstance.